### PR TITLE
Add my github profile in data.json

### PR DESCRIPTION
**Pull request by Anjan50**
Add my github profile in data.json

### CODE REVIEW

**Comment by AllanRegush** [comment]
Thank you for the PR. 👍 Could you put your profile at the end?

**Comment by Anjan50** [comment]
> Thank you for the PR. 👍 Could you put your profile at the end?

updated

File: data.json [hunk 1]
```diff
@@ -365,4 +365,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/53856673/102192030-10423b00-3e6f-11eb-8494-a684cebb2ecc.png",
     "issueNumber": 186
   }
```
Comment (AllanRegush) [inline data.json:367]: Looks like there is a missing comma
```diff
- }
+ },
```

**Review by AllanRegush — COMMENTED** [review]
Looking Good

File: data.json (hunk 1 above)
Comment (Anjan50) [reply data.json]: Updated

**Comment by Anjan50** [comment]
> Looking Good
Now, Can you merge my PR?

**Comment by eddiejaoude** [comment]
I resolved the conflicts

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Pull request merged**